openedx/core/djangoapps/content_tagging/rest_api/v1/tests/test_objecttag_export_helpers.py

```diff
@@ -0,0 +1,189 @@
+"""
+Test the objecttag_export_helpers module
+"""
+from unittest.mock import patch
+
+from openedx_tagging.core.tagging.models import Tag
+from organizations.models import Organization
+
+from common.djangoapps.student.tests.factories import UserFactory
+from xmodule.modulestore.tests.django_utils import TEST_DATA_SPLIT_MODULESTORE, ModuleStoreTestCase
+
+from .... import api
+from ..objecttag_export_helpers import TaggedContent, build_object_tree_with_objecttags, iterate_with_level
+
+
+class TaggedCourseMixin(ModuleStoreTestCase):
+    """
+    Mixin with a course structure and taxonomies
+    """
+    MODULESTORE = TEST_DATA_SPLIT_MODULESTORE
+
+    def setUp(self):
+        super().setUp()
+        # Create user
+        self.user = UserFactory.create()
+        self.user_id = self.user.id
+
+        self.orgA = Organization.objects.create(name="Organization A", short_name="orgA")
+        self.taxonomy_1 = api.create_taxonomy(name="Taxonomy 1")
+        api.set_taxonomy_orgs(self.taxonomy_1, all_orgs=True)
+        Tag.objects.create(
+            taxonomy=self.taxonomy_1,
+            value="Tag 1.1",
+        )
+        Tag.objects.create(
+            taxonomy=self.taxonomy_1,
+            value="Tag 1.2",
+        )
+
+        self.taxonomy_2 = api.create_taxonomy(name="Taxonomy 2")
+        api.set_taxonomy_orgs(self.taxonomy_2, all_orgs=True)
+
+        Tag.objects.create(
+            taxonomy=self.taxonomy_2,
+            value="Tag 2.1",
+        )
+        Tag.objects.create(
+            taxonomy=self.taxonomy_2,
+            value="Tag 2.2",
+        )
+
+        self.patcher = patch("openedx.core.djangoapps.content_tagging.tasks.modulestore", return_value=self.store)
+        self.addCleanup(self.patcher.stop)
+        self.patcher.start()
+
+        # Create course
+        self.course = self.store.create_course(
```

Comment:
nit: Recommend using CourseFactory here (and BlockFactory below) instead of the modulestore create methods.
cf https://github.com/open-craft/edx-platform/pull/622/commits/548d57c34104f54698d276c7d1d9191094f3877e